lib/zuora/soap/calls/create.rb

```diff
@@ -0,0 +1,26 @@
+module Zuora
+  module Soap
+    module Calls
+      module Create
+        # Generates a SOAP envelope for given Zuora object
+        # of `type`, having `fields`, with `data`
+        # @params [String] token
+        # @params [Symbol] type e.g. :BillRun, :Refund
+        # @params [Array] fields - hash of whitelisted zuora object field names
+        # @return [Nokogiri::Xml::Builder] - SOAP envelope
+        def self.xml_builder(token, type, data)
```

Comment:
I'm planning to cover every public interface with tests before releasing, but I haven't settled on the interface yet. For example with Subscribe, there are numerous params, and I think it will need to be a class. For now I'd like to keep integration specs only, and plan to circle back once the interfaces solidify.